Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM ubuntu:bionic as source
+
+# Set frontend mode as noninteractive (default answers to all questions)
+ENV DEBIAN_FRONTEND noninteractive
```

Comment:
I like to use DEBIAN_FRONTEND=noninteractive directly on the line before apt-get install, because if it's configured like this globally at the beginning, it will remain set even when the container is running, so even if you later `docker exec` into the container and want to `apt-get install` something, apt will think you are still non-interactive.

Reply:
> hmm, seems valid point. I'll revert the changes.

> I like to use DEBIAN_FRONTEND=noninteractive directly on the line before apt-get install, because if it's configured like this globally at the beginning, it will remain set even when the container is running, so even if you later `docker exec` into the container and want to `apt-get install` something, apt will think you are still non-interactive.